### DISCUSSION TITLE
Add keyword arguments when using extract method

### DISCUSSION THREAD

**Discussion by MartinBernstorff** [microsoft/pylance-release#4191] [Enhancement]
E.g. for code like this:

```python
bar = 4

# Extracted method
def bar(bar_kw):
    test = bar_kw*4

bar(bar)
```

I would love for the refactoring to call the function with keyword arguments:

```python
bar = 4

# Extracted method
def bar(bar_kw):
    test = bar_kw*4

bar(bar_kw=bar)
#    ^^^
```

This makes the refactoring stable to a change in the ordering or number of arguments. I recognise not everyone might want this, so perhaps an option?

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
